src/routes/oauth2/logout.ts

```diff
@@ -0,0 +1,79 @@
+import { Env } from "../../types";
+import { getClient } from "../../services/clients";
+import {
+  getStateFromCookie,
+  serializeClearCookie,
+} from "../../services/cookies";
+import { validateRedirectUrl } from "../../utils/validate-redirect-url";
+import { HTTPException } from "hono/http-exception";
+import { OpenAPIHono, createRoute, z } from "@hono/zod-openapi";
+
+export const logoutRoutes = new OpenAPIHono<{ Bindings: Env }>()
+  // --------------------------------
+  // GET /logout
+  // --------------------------------
+  .openapi(
+    createRoute({
+      tags: ["oauth2"],
+      method: "get",
+      path: "/",
+      request: {
+        query: z.object({
+          client_id: z.string(),
+          returnTo: z.string().optional(),
+        }),
+        header: z.object({
+          cookie: z.string().optional(),
+        }),
+      },
+      responses: {
+        302: {
+          description: "List of tenants",
```

Comment:
doesn't seem right to me

Reply:
```suggestion
          description: "Log the user out",
```
:question: @markusahlstrand

Reply:
I'll do this one and merge this one as it seems fine!